cmd/bootstrapper/uninstall_kubelet.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/openshift/windows-machine-config-bootstrapper/pkg/bootstrapper"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+var (
+	// uninstallKubeletCmd describes the uninstall-kubelet command
+	uninstallKubeletCmd = &cobra.Command{
+		Use:   "uninstall-kubelet",
+		Short: "Uninstalls kubelet from the Windows node",
+		Long: "Uninstalls kubelet from the Windows node. " +
+			"This command needs to be executed every time kubelet needs to be uninstalled",
+		Run: runUninstallKubeletCmd,
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(uninstallKubeletCmd)
+}
+
+// runUninstallKubeletCmd uninstalls kubelet service from the Windows node
+func runUninstallKubeletCmd(cmd *cobra.Command, args []string) {
```

Comment:
I suggest following the existing pattern of:
```go
bootstrapper.NewWinNodeBootstrapper()
wmcb.UninstallKubelet()
wmcb.Disconnect()
```
Otherwise you are leaking implementation details here.

Reply:
I agree that was my initial thought but there would be a lot of parameters in calling bootstrapper.NewWinNodeBootstrapper() which would be not be required for uninstall eg: k8sInstallDir, ignitionFile, kubeletPath and would have to be passed as empty strings.

Reply:
Think of WMCB as a library and how a user would call. This makes it way too complicated from that perspective. Having empty strings is not a reason for doing this. We can document that the inputs will be ignored if using the uninstall kubelet functionality.

Reply:
that works.